default/scripting/techs/construction/TRANSCEND_ARCH.focs.txt

```diff
@@ -10,7 +10,11 @@ Tech
         "CON_ORGANIC_STRC"
         "CON_NDIM_STRC"
     ]
-    unlock = Item type = Building name = "BLD_MEGALITH"
+    unlock = 
```

Comment:
Didn't notice earlier or something went weird with merging, but this extra line is a problem...

Reply:
Oops. Yes, that was wrong in my PR, sorry.